av1an-core/src/concat.rs

```diff
@@ -197,54 +197,10 @@ pub fn mkvmerge(
     .map(|chunk| PathBuf::from(format!("{:05}.{}", chunk, encoder.output_extension())))
     .collect();
 
+  let mut this_file: File = File::create("options.json").unwrap();
```

Comment:
We probably want to create this file in the temp folder.

```suggestion
  let options_path = PathBuf::from(&temp_dir).join("options.json");
  let mut this_file: File = File::create(&options_path).unwrap();
```